pytest/tests/loadtest/locust/locustfile.py

```diff
@@ -34,3 +39,53 @@ def on_start(self):
         logger.debug(
             f"{self.account_id} ready to use FT contract {self.ft.account.key.account_id}"
         )
+
+
+class SocialDbUser(NearUser):
```

Comment:
Good catch!

The idea with multiple locust users is to select them as CLI argument. (e.g. `locust FTTransferUser` will only run tasks for that user)
But that's not quite enough for us, as we have initialization code in the common directory that must run depending on which user classes are enabled.

Locust also has tags on tasks that can be used to further specify which tasks to run. (example: `--tag ft social`) Sadly, tags are not further integrated. We can't use them on events, for example.

Nevertheless, I have now changed the code so that the WASM arguments are only required when we actually need them. For that, I slightly misused tags by introducing an `ft` tag that enables all the FT initialization code using manual checks if the tag is enabled. Similar for the `social` tag. (The tags are also on the corresponding tasks for cosmetic reasons but locust will spawn such users even when they have no active task so we do need to select the user class even if we already selected the tag.)

This setup is a bit unfortunate because one has to select both `FTTransferUser` and `--tag ft` for it to work properly. But I don't want the `ft.py` to have a backwards dependency to user classes defined in locustfile.py as this would lead to problems when we introduce multiple locustfiles. So that's unfortunately the best solution that I could find so far.

But of course one can still run everything and in that case one doesn't have to worry about tags / user classes.

I have also updated the README accordingly.